netutils/webserver/httpd_dirlist.c

```diff
@@ -193,11 +193,10 @@ ssize_t httpd_dirlist(int outfd, FAR struct httpd_fs_file *file)
           break;
         }
 
-      path = malloc(CONFIG_NAME_MAX);
+      path = malloc(2 * PATH_MAX);
```

Comment:
```suggestion
      path = malloc(sizeof(file->path) + sizeof(dent->d_name) + 1);
```

maybe we can even use `ret` to:
```
      ret = sizeof(file->path) + sizeof(dent->d_name) + 1;
      path = malloc(ret);

      snprintf(path, ret, "%s/%s", file->path, dent->d_name);
```

Reply:
I change to asprintf instead.